app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -261,4 +285,21 @@ public void markMessageAsRead(Message message) {
             unreadMessageHandler.sendEmptyMessageDelayed(0, 2000);
         }
     }
+    public void muteTopic(Message message) {
+        mutedTopics.add(message.getStream().getId() + message.getSubject());
```

Comment:
I'd still prefer to have the operation for generating the 'key' for a muted topic to be a separate function, or even embed it into the `Message` object. for example:

    mutedTopics.add(message.concatStreamAndTopic());
    ...
    // in Message.java
    private String concatStreamAndTopic() {
        return getStream().getId() + getSubject();
    }